src/utils/api.js

```diff
@@ -8,12 +8,20 @@ export const getReviews = async (category) => {
   if (!category) {
     const { data } = await gamesApi.get("/reviews");
     return data.reviews;
-  } else if (category) {
+  }
+  if (category) {
     const { data } = await gamesApi.get(`/reviews?category=${category}`);
     return data.reviews;
   }
 };
 
+export const getSortedReviews = async (sort_by, order) => {
+  const { data } = await gamesApi.get("/reviews", {
+    params: { sort_by, order },
```

Comment:
Love the axios params here, prevents building this string manualy with loads of if statements!